components/apimgt/org.wso2.carbon.apimgt.core/src/test/resources/testngDAO.xml

```diff
@@ -10,6 +10,7 @@
             <class name="org.wso2.carbon.apimgt.core.dao.impl.LabelDAOImplIT" />
             <class name="org.wso2.carbon.apimgt.core.dao.impl.WorkflowDAOIT" />
             <class name="org.wso2.carbon.apimgt.core.dao.impl.PolicyDAOImplIT" />
+            <class name="org.wso2.carbon.apimgt.core.dao.impl.FunctionDAOImplIT" />
```

Comment:
We are testing the SQL queries in each DAO with different types of DBs. So can't we consider them as Integration Tests?